customer-commons/src/main/java/no/unit/nva/customer/model/CustomerReference.java

```diff
@@ -38,11 +37,11 @@ public void setDisplayName(String displayName) {
     }
 
     public String getCreatedDate() {
-        return instantToString(createdDate);
```

Comment:
are you sure we can refactor this? instantToString does the same as toString without special formatting?